### PR TITLE
[CI]add dead link check

### PR DESCRIPTION
**Pull request by CalvinKirs**

### CODE REVIEW

File: .github/workflows/dead-link-checker.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+jobs:
+  CheckDeadLinks:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v2
+      - run: sudo npm install -g markdown-link-check
+      - run: |
+          for file in $(find . -name "*.md"); do
+            markdown-link-check -c .dlc.json -q "$file"
```
Comment (kezhenxu94) [inline .github/workflows/dead-link-checker.yaml:35]: You missed a file https://github.com/apache/skywalking/blob/master/.dlc.json

**Comment by zhongjiajie** [comment]
Should we also add it to ds-website repo?

File: .github/workflows/dead-link-checker.yaml (hunk 1 above)
Comment (CalvinKirs) [reply .github/workflows/dead-link-checker.yaml:35]: well, thanks :)

**Comment by CalvinKirs** [comment]
> Should we also add it to ds-website repo?

yup

**Comment by caishunfeng** [comment]
Hi @CalvinKirs Is this PR still useful?

**Comment by CalvinKirs** [comment]
> Hi @CalvinKirs Is this PR still useful?

Thx,I will reopen  after finishing it.

**Pull request closed without merging**